source/docs/tuner/device-list.rst

```diff
@@ -74,10 +74,14 @@ Once the upgrade dialog is opened, information detailing the device name, model,
    :width: 70%
    :alt: Batch selection screen
 
-.. note:: The :guilabel:`Pro` column has no effect if you're using the update offline button.
+.. note:: The :guilabel:`Pro` column has no effect if you're using the "Select firmware..." button.
```

Comment:
I don't know how I missed this, but this is actually wrong now. The `Pro` column does affect "Select firmware...", as we have separate popups for Pro and v5.